### ISSUE TITLE
Changing chat font size

### ISSUE THREAD

**Issue by nauposerver** [transhumandesign/kag-base#371]
There should be an option for changing chat font size, the current size is very very small to me, I always have to lean in to see better.

**Comment by AsuMagic** [comment]
Agreed, however the font rendering in KAG is fairly rigid as is. I added the `engine change` label because I feel like this might require some tinkering in the engine side to actually render properly if we ever do this.

**Comment by asumagic** [comment]
Closing because if it is ever addressed, it will be under the form of render scales rather than font size most likely.
